include/Megamix/Region.hpp

```diff
@@ -23,6 +23,8 @@ namespace Region {
     std::vector<u32> RHMPatchGameAddresses();
     std::vector<u32> RHMPatchTempoAddresses();
     std::vector<u32> RHMPatchGateAddresses();
+    std::vector<u32> MuseumRowsInfoAddresses();
+    std::vector<u32> MuseumRowsColorsAddresses();
```

Comment:
Add a blank line between the RHMPatch and MuseumRows stuff, otherwise it looks like the same category